### PR TITLE
Add tests for model serving with gpus

### PR DESCRIPTION
**Pull request by lugi0**

### CODE REVIEW

**Comment by lugi0** [comment]
Pr test: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1819/console 
All Pass

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/421__model_serving_gpu.robot [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+Test Inference Load On GPU
+    [Documentation]    Test the inference load on the GPU after sending random requests to the endpoint
+    [Tags]    Sanity    Tier1    Resources-GPU
+    ...    ODS-2213
+    ${url}=    Get Model Route via UI    ${MODEL_NAME}
+    Send Random Inference Request     endpoint=${url}    no_requests=100
+    # Verify metric DCGM_FI_PROF_GR_ENGINE_ACTIVE goes over 0
+    ${prometheus_route} =    Get OpenShift Prometheus Route
+    ${sa_token} =    Get OpenShift Prometheus Service Account Token
+    ${expression} =    Set Variable    DCGM_FI_PROF_GR_ENGINE_ACTIVE  #&step=1
```
Comment (bdattoma) [inline ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/421__model_serving_gpu.robot:63]: is `#&step=1` needed?

**Review by bdattoma — APPROVED** [review]

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/421__model_serving_gpu.robot (hunk 1 above)
Comment (lugi0) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/421__model_serving_gpu.robot:63]: I'm not 100% sure, in the documentation I found it's supposed to return the last time step available with that appended, but even like this I only get 1 return value which seems correct to me

Comment (bdattoma) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/421__model_serving_gpu.robot:63]: hm I think RF may not read correctly this line of code. In the html log I see:
`${expression} = DCGM_FI_PROF_GR_ENGINE_ACTIVE`

Comment (bdattoma) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/421__model_serving_gpu.robot:63]: oh I thought the "#" was supposed to be part of the query, sorry :)

Comment (lugi0) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/421__model_serving_gpu.robot]: Removed it since it's not needed currently

**Review by bdattoma — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]

**Pull request merged**